fhireval/data/cmg-condition-disease.json

```diff
@@ -0,0 +1,415 @@
+{
+  "groupname": "CMG-Examples",
+  "resources": {
+    "Condition": [
+      {
+        "resourceType": "Condition",
+        "identifier": [
+          {
+            "system": "urn:ncpi:unique-string",
+            "value": "FHIR-CMG-TEST|ct-fam1|ct.patient01|ARTHROGRYPOSIS, DISTAL, TYPE 1A; DA1A"
```

Comment:
Didn't you mention Google Healthcare API doesn't allow to query with string including `|`?